### PR TITLE
[Team 1 Add Experience] Added Profile step definition class with some steps, addExperience.feature file

### PR DESCRIPTION
**Pull request by tylee17**
Test passed

### CODE REVIEW

File: src/test/java/uiStepDefinitions/Profile.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	}
+
+	@When("user enters experiences")
+	public void user_enters_experiences(io.cucumber.datatable.DataTable dataTable) {
+		Map<String, String> experience = dataTable.asMaps().get(0);
+		driver.findElement(By.xpath("//input[@placeholder='* Job Title']")).sendKeys(experience.get("jobtitle"));
+		driver.findElement(By.name("company")).sendKeys(experience.get("company"));
+		driver.findElement(By.name("location")).sendKeys(experience.get("company"));
+		driver.findElement(By.name("from")).sendKeys(experience.get("from"));
+		driver.findElement(By.xpath("//input[@name='current']")).click();
```
Comment (Muradil-Erkin) [inline src/test/java/uiStepDefinitions/Profile.java]: This step will not handle the `not current` situation

File: src/test/java/runners/SmokeRunner.java [hunk 2]
```diff
@@ -13,6 +13,6 @@
 @ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "pretty, html:target/html-report/index.html")
 @ConfigurationParameter(key = GLUE_PROPERTY_NAME, value = "uiStepDefinitions, apiStepDefinitions, hooks")
 @ConfigurationParameter(key = ANSI_COLORS_DISABLED_PROPERTY_NAME, value = "true")
-@ConfigurationParameter(key = FILTER_TAGS_PROPERTY_NAME, value = "@smoke")
+@ConfigurationParameter(key = FILTER_TAGS_PROPERTY_NAME, value = "@smoke and @addExperience")
```
Comment (Muradil-Erkin) [inline src/test/java/runners/SmokeRunner.java]: Please revert changes to this file

**Review by Muradil-Erkin — CHANGES_REQUESTED** [review]

**Review by Muradil-Erkin — APPROVED** [review]
LGTM!

**Review by qingyuanre1 — APPROVED** [review]
approve

**Pull request merged**